backend/src/Duxcore.ts

```diff
@@ -0,0 +1,38 @@
+import { config as dtCfg } from 'dotenv'
+import { SessionServer } from './classes/SessionServer';
+import keypair from 'keypair';
+import { WebSocketServer } from './classes/WebSocketServer';
+
+export default class Duxcore {
+	
+	public tmpKeypair = keypair();
```

Comment:
using the tab symbol?

Reply:
wdym?

Reply:
in [.prettierrc.js](https://github.com/HoloPanio/duxcore/blob/dev/.prettierrc.js) `useTabs: false` and you are using tabs rather than spaces